crates/schemalate/src/firebolt/firebolt_projections.rs

```diff
@@ -101,6 +107,14 @@ pub fn validate_new_projection(
                             "The root document is usually not necessary in delta-update connectors."
                                 .to_string(),
                     }
+                } else if !VALID_FIELD_REGEX.is_match(&projection.field) {
+                    Constraint {
+                        r#type: constraint::Type::FieldForbidden.into(),
+                        reason: format!(
+                            "Field names must match the regex {}.",
```

Comment:
To avoid user confusion, please clarify that this isn't a Flow restriction, but is a Firebolt one. E.g. "Firebolt requires that... "